tests/unittests/OperatorTest.cpp

```diff
@@ -6986,18 +6986,17 @@ TEST_P(OperatorTest, SigmoidCrossEntropyWithLogits) {
 TEST_P(OperatorTest, insertTensorTest) {
   ENABLED_BACKENDS(Interpreter, CPU);
```

Comment:
nit: seems like OpenCL also has InsertTensor instr support, could just add that as well.

Reply:
OpenCL has insertTensor but only for FP32 and Int8 - this test covers Int64. So it's right to be disabled.

The concatTests optimize down to InsertTensors so the OCL code path is covered by OperatorTests.